seedlet/README.md

```diff
@@ -15,93 +15,93 @@ When you reduce a theme design down to a set of systematic design decisions, you
 
 **Editor** 
-- `/varya/variables-editor.css`
-- `/varya/style-editor.css`
+- `/seedlet/variables-editor.css`
+- `/seedlet/style-editor.css`
 
 In each view, the variables are loaded first and then the stylesheet is loaded which applies the variables.
 
 ## Quick start guide
 
 1. Clone / download this repository into the `wp-content/themes` directory of your WordPress site.
-2. `cd` into the `wp-content/themes/varya` directory and run `npm install`.
-3. Next, run `npm build child-theme` and go through the prompts. This will create a fresh Varya-based child-theme using the name you chose in the prompts (*The build tool is a work in progress. Follow the manual directions below to complete this step).
+2. `cd` into the `wp-content/themes/seedlet` directory and run `npm install`.
+3. Next, run `npm build child-theme` and go through the prompts. This will create a fresh Seedlet-based child-theme using the name you chose in the prompts (*The build tool is a work in progress. Follow the manual directions below to complete this step).
 4. Visit the WordPress admin, access the Themes page and activate the new child-theme. 
 5. Then, visit the frontend of your site and to see how it looks by default.
 6. Next, open `child-theme/variables.css` in a text editor, change one or more of the variables, save the file and then refresh the front end of your site to see how it affects the design. 
-7. Continue to add and/or remove variables based on the [full list of available variables in Varya](https://github.com/Automattic/themes-workspace/blob/add/variatheme/varya/variables.css).
+7. Continue to add and/or remove variables based on the [full list of available variables in Seedlet](https://github.com/Automattic/themes-workspace/blob/add/variatheme/seedlet/variables.css).
 8. Once you’re variables are all set you can continue development like you would with any child-theme using `child-theme/style.css` to add any extra CSS you might need to accomplish a design. 
 
 #### Create a child-theme manually (*optional)
 
-To use the system, simply duplicate the `varya-child` theme directory and rename it `my-theme-name`. You’ll also want to do a search for `varia-child` strings and replace them with `my-theme-name` as follows:
+To use the system, simply duplicate the `seedlet-child` theme directory and rename it `my-theme-name`. You’ll also want to do a search for `varia-child` strings and replace them with `my-theme-name` as follows:
 
-- Search for: `'varya-child'` and replace with: `'my-theme-name'` (with quotes).
-- Search for: `varya_child_` and replace with: `my_theme_name_`.
+- Search for: `'seedlet-child'` and replace with: `'my-theme-name'` (with quotes).
+- Search for: `seedlet_child_` and replace with: `my_theme_name_`.
 - Search for: `Text Domain: varia-child` and replace with: `Text Domain: my-theme-name` in _style.css_.
 - Search for:  `varia-child` and replace with: `my_theme_name`.
 - Search for: `varia-child-` and replace with: `my-theme-name-`.
 
```

Comment:
A few instances of `varia-` popped up here. 
Otherwise, I think this is good to go :-)

Reply:
Oh weird! I wonder how that got through the find & replace. 🤔 

Thanks for catching!

Reply:
Should be fixed in 4b73264. Mind taking one more look?